src/core/utils/TextTruncation.tsx

```diff
@@ -0,0 +1,44 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+
+export type TextTruncationProps = {
+  /**
+   * Text to truncate.
+   */
+  text: string;
+  /**
+   * Number of characters to leave at the end.
+   * @default 6
+   */
+  endCharsCount?: number;
+};
```

Comment:
If we expose it public use, maybe extent `CommonProps`.

Reply:
added